test/recipes/20-test_dhparam.t

```diff
@@ -36,11 +36,13 @@ sub checkdhparams {
         #Text file. Check it looks like PEM
         open(PEMFILE, '<', $file) or die $!;
         if (my $firstline = <PEMFILE>) {
-            chomp($firstline);
+            $firstline =~ s/\r|\n//g;
```

Comment:
The most *proper* is what we have in a number of other places:

``` perl
$blah =~ s/\R$//;
```

yes, perlre has a specific special escape to match any line ending.  Here's what `perlrebackslash(1)` has to say:

```
       \R  "\R" matches a generic newline; that is, anything considered a
           linebreak sequence by Unicode. This includes all characters matched
           by "\v" (vertical whitespace), and the multi character sequence
           "\x0D\x0A" (carriage return followed by a line feed, sometimes
           called the network newline; it's the end of line sequence used in
           Microsoft text files opened in binary mode). "\R" is equivalent to
           "(?>\x0D\x0A|\v)".  (The reason it doesn't backtrack is that the
           sequence is considered inseparable.  That means that

            "\x0D\x0A" =~ /^\R\x0A$/   # No match

           fails, because the "\R" matches the entire string, and won't
           backtrack to match just the "\x0D".)  Since "\R" can match a
           sequence of more than one character, it cannot be put inside a
           bracketed character class; "/[\R]/" is an error; use "\v" instead.
           "\R" was introduced in perl 5.10.0.

           Note that this does not respect any locale that might be in effect;
           it matches according to the platform's native character set.

           Mnemonic: none really. "\R" was picked because PCRE already uses
           "\R", and more importantly because Unicode recommends such a
           regular expression metacharacter, and suggests "\R" as its
           notation.
```

Reply:
Fixed - and confirmed that the amended code works as expected with mingw and in Linux.